scripts/reporting/release_to_synapse

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+##
+##  Copyright 2015 SRI International
+##  License: https://ncanda.sri.com/software-license.txt
+##
+"""
+release_to_synapse
+============================================
+This script uploads release csvs to synapse
+"""
+
+import os
+from os import listdir
```

Comment:
generally with `import os` you just access these functions using `os.listdir` or `os.path.isfile` rather than using the `from` statement